displayLinac.py

```diff
@@ -120,6 +120,11 @@ def create_faults(self):
                 pv = self.pv_addr(suffix)
 
             elif level == "CM":
+                cm_type = csvFaultDict["CM Type"]
```

Comment:
Debugging currently, but I suspect we need to define the prefix (similar to what we did in the cryo level snippet of code a few lines up)